### PR TITLE
Fix/daily metrics capture non prod envs

### PR DESCRIPTION
**Pull request by haraprasadj**
Introduced an optional parameter "ns" to reports-cronjob to enable getting logs mapped to specific namespace. The default value is "default", which does not help capture metrics from vpc's with multiple k8s namespaces like the qa env's.

### CODE REVIEW

File: files/scripts/reports-cronjob.sh [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #
 # PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 # vpc_name=YOUR-VPC-NAME
+# ns=KUBERNETES_NAMESPACE (optional, defaults to "default")
 # USER=USER
 # KUBECONFIG=path/to/kubeconfig
 # 3   3   *   *   *    (if [ -f $HOME/cloud-automation/files/scripts/reports-cronjob.sh ]; then bash $HOME/cloud-automation/files/scripts/reports-cronjob.sh "vpc=$vpc_name"; else echo "no reports-cronjob.sh"; fi) > $HOME/reports-cronjob.log 2>&1
```
Comment (jawadqur) [inline files/scripts/reports-cronjob.sh:11]: Can you also update the reports-cronjob to have ns=$ns as arg?

File: files/scripts/reports-cronjob.sh [hunk 2]
```diff
@@ -26,7 +27,7 @@ source "${GEN3_HOME}/gen3/gen3setup.sh"
 
 help() {
   cat - <<EOM
-Use: bash ./reports-cronjob.sh vpc=YOUR_VPC_NAME
+Use: bash ./reports-cronjob.sh vpc=YOUR_VPC_NAME [ns_name=K8S_NAMESPACE]
```
Comment (jawadqur) [inline files/scripts/reports-cronjob.sh:30]: can you remove the brackets?

**Review by jawadqur — APPROVED** [review]

File: files/scripts/reports-cronjob.sh (hunk 2 above)
Comment (haraprasadj) [reply files/scripts/reports-cronjob.sh:30]: oh, that is just to indicate it is optional :)

Comment (jawadqur) [reply files/scripts/reports-cronjob.sh:30]: 👍

**Pull request merged**